### PR TITLE
Clean up design tokens pre implemented

### PR DESCRIPTION
**Pull request by tilucasoli**
## Describe your changes
- Remove pre-implemented Space and Radius Tokens
- Remove tokens utilities for Space and Radius
- Rename space to spaces
## Type of change

- **New feature**
- **Test**

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/src/theme/mix/mix_tokens.dart [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  final textStyle = const _MixTextStyles();
+  final space = const _MixSpaces();
+  final radius = const _MixRadii();
+
+  const MixTokens();
+}
+
+@immutable
+class _MixColorTokens {
+  final primary = const ColorToken('mix.color.primary');
```
Comment (leoafarias) [inline lib/src/theme/mix/mix_tokens.dart:20]: These would not be Mix Tokens but Material ones. I don't think we should use them as MixTokens by default but maintain the ability to import material tokens into it.

**Review by leoafarias — CHANGES_REQUESTED** [review]

File: lib/src/theme/tokens/radius_token.dart [hunk 2]
```diff
@@ -58,27 +50,3 @@ class RadiusRef extends Radius with TokenRef<RadiusToken, Radius> {
   @override
   int get hashCode => token.hashCode;
 }
-
-// // Helper class to wrap functions that can return
-// a Radius token that resmebles the WithSpaceToken
-@immutable
-class UtilityWithRadiusTokens<T> {
```
Comment (leoafarias) [inline lib/src/theme/tokens/radius_token.dart:65]: Are the other utilities still within Mix? I would like to maintain this here, to help compose utilities.

**Review by leoafarias — CHANGES_REQUESTED** [review]
LGTM

File: lib/src/theme/tokens/radius_token.dart (hunk 2 above)
Comment (tilucasoli) [reply lib/src/theme/tokens/radius_token.dart:65]: Leo, are you sure about this? This class wasn't instantiated anywhere in the code.

File: lib/src/theme/mix/mix_tokens.dart (hunk 1 above)
Comment (tilucasoli) [reply lib/src/theme/mix/mix_tokens.dart]: done

**Pull request merged**